src/main/resources/swagger.api/schemas/transactionStatus.yaml

```diff
@@ -0,0 +1,18 @@
+TransactionStatus:
+  type: object
+  properties:
+    status:
+      type: string
+      enum:
+        - CREATED #Created by DCB
+        - OPEN #Created in lending library, Item is available
+        - CANCELLED #Request canceled by lending library or patron
+        - IN_TRANSIT #checked in at another service point
```

Comment:
@kalibek-epam 
Does this state IN_TRANSIT and IN_TRANSIT_TO_LENDING still exists and be present in the yaml file?